Pegasus/include/Math.hpp

```diff
@@ -30,13 +659,450 @@ class JacobiEigenvalue
     glm::dmat3 m_eigenvectors;
     glm::dvec3 m_eigenvalues;
 
+    /**
+     * @brief Finds maximal off diagonal matrix element.
```

Comment:
You have some `Norm` in method signature but not in method description. Is it intended?